components/messageBox/messageBox.js

```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes } from 'react';
+import { getClassNamesWithMods, getDataAttributes } from '../_helpers';
+
+const MessageBox = (props) => {
+  const { children, dataAttrs, isError } = props;
```

Comment:
is it possible to add `title`?
![screenshot at 09-43-41](https://user-images.githubusercontent.com/5457280/28657481-07a13264-72b0-11e7-86ae-33c308515bd5.png)